test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const data = {
+    name: 'John',
+    age: '31',
+    city: 'New York'
+};
+
+describe('First Api Tests', () => {
+    it('Consume GET Service', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+    it('Consume GET Service with query parameters', async () => {
+        const response = await axios.get('https://httpbin.org/get', data);
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(data);
```

Comment:
as we removed the brackets, it should be `expect(response.config).to.eql(data);`